plugins/inventory/cloudstack.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Rafael del valle <rafael@privaz.io>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import sys
+import os
+import traceback
+import yaml
+
+DOCUMENTATION = r'''
+    name: cloudstack
+    plugin_type: inventory
+    short_description: Apache CloudStack inventory source
+    author: Rafael del Valle (@rvalle)
+    version_added: 2.1.0
+    description:
+        - Get inventory hosts from Apache CloudStack
+        - Allows filtering and grouping inventory hosts.
+        - |
+            Uses an YAML configuration file ending with either I(cloudstack.yml) or I(cloudstack.yaml)
+            to set parameter values (also see examples).
+    options:
+        plugin:
+            description: Token that ensures this is a source file for the 'cloudstack' plugin.
+            type: string
+            required: True
+            choices: [ cloudstack ]
```

Comment:
extend choices by `ngine_io.cloudstack.cloudstack` this probably fixes the build